src/index.html

```diff
@@ -17,6 +25,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO Logo"
+          class="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
```

Comment:
Avoid using spaces in the `href` attribute of the `<a>` tag. If this is meant to be a placeholder, consider using '#apple' to indicate the intended target section.